CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # Basic CMake Setup
 #------------------------------------------------------------------------------
 
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Some of the host_configs for me were set to an older version of cmake, and this was just a quick workaround.